db/migrate/20150908222711_drop_marketing_engineering_products.rb

```diff
@@ -0,0 +1,20 @@
+class DropMarketingEngineeringProducts < ActiveRecord::Migration
+  class Katello::MarketingProduct < ActiveRecord::Base
+    self.table_name = "katello_products"
+  end
+
+  def up
+    drop_table :katello_marketing_engineering_products
+
+    remove_column :katello_products, :type
+  end
```

Comment:
I may have missed it, but i don't see you removing the old marketing products prior to dropping the 'type' column?